jupyter/rocm/tensorflow/ubi9-python-3.11/Pipfile

```diff
@@ -4,17 +4,17 @@ verify_ssl = true
 name = "pypi"
 
 [dev-packages]
+tf2onnx = "~= 1.16.1"
```

Comment:
I'd like to see a comment here, why tf2onnx is in dev-packages

Reply:
Added details now.
let me know if that fits.

Reply:
thanks, looks informative enough to me!